src/CMakeLists.txt

```diff
@@ -97,6 +100,9 @@ IF(NIMBLE_HAVE_MPI)
     )
   add_executable(NimbleSM_MPI nimble_mpi_main.cc)
   target_link_libraries(NimbleSM_MPI PRIVATE nimble::nimble)
+  if (NIMBLE_HAVE_KOKKOS)
```

Comment:
The same change is required here, linking Trilinos::Trilinos instead of Kokkos::kokkos if I have Trilinos enabled.

Reply:
Done.